scripts/src/java/org/oppia/android/scripts/xml/BUILD.bazel

```diff
@@ -0,0 +1,48 @@
+"""
+Package for Xml syntax based checks.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_library", "kt_jvm_test")
+
+kt_jvm_library(
+    name = "xml_syntax_error_handler_lib",
+    testonly = True,
+    srcs = ["XmlSyntaxErrorHandler.kt"],
```

Comment:
Why not just include this in xml_syntax_check_lib? Will it ever be used anywhere else?

Reply:
Ah! this lib is also used in `XmlSyntaxErrorHandlerTest`, so that's why I decided to separate it out.

Reply:
Got it. In that case, let's strike "_lib" from the library name since it's not needed here. We only use that for the root script libraries since they have binary targets with the same name which could introduce confusion in the build graph. By default, we don't include "_lib" since it's redundant.

Reply:
Done.